src/services/__tests__/time-entries-api.test.js

```diff
@@ -0,0 +1,33 @@
+const fetchTimeEntries = require('../time-entries-api');
+
+const mockedTimeEntries = [{
+  client:{
+    branch: '123',
+    clientName: '123',
+    contactPerson: 'test',
+    contactPhoneNumber: 'test',
+    id: 123,
+    locality: '123',
+    postalCode: '123'
+  },
+  id: 123,
+  startTimestamp: 'bla',
+  stopTimestamp: 'bla'
+}]
+
+it('fetches TimeEntries from the server', () => {
+  const mockSuccessResponse = {mockedTimeEntries};
+  const mockJsonPromise = Promise.resolve(mockSuccessResponse);
```

Comment:
Are you aware that the `mockSuccessResponse` looks like this:

```
{
  mockedTimeEntries: [
    {
      // properties
    }
  ]
}
```
That doesn't seem to make sense. Why don't you just resolve like this: `Promise.resolve(mockedTimeEntries)`.

Reply:
you're right! adjusted